.github/workflows/continuous-integration.yml

```diff
@@ -36,3 +36,36 @@ jobs:
 
       - name: "Psalm"
         run: "vendor/bin/psalm"
+
+  coding-standards:
+    name: "Coding Standards"
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
```

Comment:
Now that you fixed the issue with 7.4 locally, maybe we can use 7.4 here too?

Reply:
well, the issue seems to be in vendors, but let's try.

Reply:
seems fine